ISLE/isleapp.h

```diff
@@ -1,15 +1,14 @@
 #ifndef ISLEAPP_H
 #define ISLEAPP_H
 
+#include "lego/legoomni/include/legoutils.h"
```

Comment:
I think it should be possible to directly include legoutils.h without the path presented?

Reply:
Yes, it's not needed.
I don't know why it did not work when I first tried to fix this.